component/common.libsonnet

```diff
@@ -31,39 +40,23 @@ local bypassNamespaceRestrictionsSubjects() =
     subjects+: flattenSet(bypass.subjects),
   };
 
-local matchNamespaces(selector=null, names=null) = {
-  all+: [ {
+local matchKinds(selector=null, names=null, match='all', kinds) = {
+  [match]+: [ {
     resources+: std.prune({
-      kinds+: [
-        'Namespace',
-      ],
+      kinds+: kinds,
       selector+: selector,
       names+: names,
     }),
   } ],
 };
 
-local matchProjectRequests(selector=null, names=null) = {
-  all+: [ {
-    resources+: std.prune({
-      kinds+: [
-        'ProjectRequest',
-      ],
-      selector+: selector,
-      names+: names,
-    }),
-  } ],
-};
+local matchNamespaces(selector=null, names=null, match='all') = matchKinds(selector, names, match, kinds=[ 'Namespace' ]);
+
+local matchProjectRequests(selector=null, names=null, match='all') = matchKinds(selector, names, match, kinds=[ 'ProjectRequest' ]);
 
 local matchOrgNamespaces = matchNamespaces(
-  selector={
-    matchExpressions: [
-      {
-        key: 'appuio.io/organization',
-        operator: 'Exists',
-      },
-    ],
-  }
+  selector=orgLabelSelector,
+  match='any',
```

Comment:
Setting this to `all` would not change anything but create less diff noise.

Reply:
Right. Setting to `any` was required when I wanted to combine ProjectRequest + namespace kind in one rule, but that doesn't work anyway.